src/client/views/components/head.ejs

```diff
@@ -6,7 +6,7 @@
 <meta charset="utf-8" />
 <meta http-equiv="x-ua-compatible" content="ie=edge" />
 <meta name="viewport" content="width=device-width, initial-scale=1" />
-<link rel="icon" type="image" href="/img/favicon.png">
+<link rel="icon" type="image" href="/img/logo.png">
```

Comment:
I don't think it's standard to have the tab icon named logo..

Reply:
A favicon is a browser icon, short for favorite icon. Many browsers will auto-request a favicon image if it exists because it's so commonly given that name. If it's named differently it's not immediately clear that it is a browser icon, there might even be extra requests, with the browser fetching logo.png *and* seeing if we have a "favicon" as well.

Reply:
We now have a new server icon!